Backpack/Card/Classes/BPKCard.h

```diff
@@ -32,6 +32,11 @@ NS_SWIFT_NAME(Card) IB_DESIGNABLE @interface BPKCard: UIControl
  */
 @property (nonatomic, assign, getter=isPadded) IBInspectable BOOL padded;
 
+/**
+ * The view to display within the card.
+ */
+@property(nonatomic, assign, nullable) UIView *subview;
```

Comment:
I was thinking, `contentView` might be a better name. Would align more with UIKit standard naming, for example on UICollectionViewCell.h:
```objc
@property (nonatomic, readonly) UIView *contentView; // add custom subviews to the cell's contentView
```

Reply:
Possibly, it needs to make sense for the divided card too